### ISSUE TITLE
Map Widget: Initial map view is too small and lack of color contrast

### ISSUE THREAD

**Issue by dan-du-car** [VolpeUSDOT/CDA1Tenth#16]
Upon launching the map UI, the initial map view is too small. It is also hard to see the white colored map with the grey background, see below:
![Image](https://github.com/user-attachments/assets/588a2926-1e21-4eca-b75a-15b3864be390)

**Steps to produce the scenario:**
- Launch the map UI using command `python3 drayageUI.py`
- Click the maximize button at the top right
